.yamllint

```diff
@@ -0,0 +1,10 @@
+---
+extends: default
+
+rules:
+    document-end:
+        present: false
+    indentation:
+        indent-sequences: false
+        spaces: 4
```

Comment:
Wouldn't `spaces: 2` match the current style better?

Reply:
The pre-existing yaml files in the repo (action.yml and .github/workflows/python-app.yml) use 2-space indent.  I'm suggesting to follow that precedent in .pre-commit-config.yaml and the yamllint config instead of switching everything over to 4 spaces.

Reply:
Got it! I'd say: let's use the same indentation everywhere for the sake of consistency. I don't mind whether that indentation is 2 or 4 spaces as long as it's the same across projects.